sci-biology/open-ephys-gui/open-ephys-gui-0.5.2.2.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit cmake
+
+DESCRIPTION="Processing, recording, and visualizing multichannel ephys data"
+HOMEPAGE="http://www.open-ephys.org/gui/"
+LICENSE="GPL-3"
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/open-ephys/plugin-GUI"
+	EGIT_BRANCH="master"
+    inherit git-r3
+	Suffix=$EGIT_BRANCH
+	SubDir=${P}
+elif [[ ${PV} == "99999999" ]] ; then
+	EGIT_REPO_URI="https://github.com/open-ephys/plugin-GUI"
+	EGIT_BRANCH="development"
+    inherit git-r3
+	Suffix=$EGIT_BRANCH
+	SubDir=${P}
+else
+    SRC_URI="https://github.com/open-ephys/plugin-GUI/archive/v${PV}.tar.gz"
+	Suffix=${PV}
+	SubDir="plugin-GUI-${PV}"
+	S="${WORKDIR}/$SubDir"
+fi
+
+SLOT="${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="jack +KWIKFormat +NetworkEvents +NWBFormat"
+
+BDEPEND="
+	<sys-devel/gcc-9
```

Comment:
can we not have nice things? :( it's ok, if we can't, but are you sure that this constraint is necessary

Reply:
Open-ephys depends on JUCE, which is bundled upstream. The current version bundled does not appear to support >gcc-8 (at least I cannot compile it in my machine unless I switch to gcc-8). I use and update this software regularly and I always try to compile with the newest gcc, so I'll for sure drop this restriction as soon as I'm able to compile it with a newer version.

Reply:
ok :)